src/common/Gateway.sol

```diff
@@ -280,7 +280,7 @@ contract Gateway is Auth, Recoverable, IGateway {
                 TransientArrayLib.push(BATCH_LOCATORS_SLOT, _encodeLocator(centrifugeId, poolId));
                 TransientBytesLib.set(batchSlot, message);
```

Comment:
Should we remove `.set()` and just use `append()` every time?

Reply:
I think as a lib, having `set()` could be beneficial (?). Not sure if you ask for saving audit lines

Reply:
Yeah I mainly ask for purpose of simpler audit exactly

Reply:
yes, done!